libcloud/compute/drivers/gce.py

```diff
@@ -38,6 +38,7 @@
 from libcloud.utils.iso8601 import parse_date
 
 API_VERSION = 'v1'
+DEFAULT_AGGREGATED_MAX_RESULTS = 500
```

Comment:
Yes, definitely the default is 500 for all the APIs.  I have conflicting views about being explicit, but I'll take it out for simplicity.